PCA/README.md

```diff
@@ -0,0 +1,24 @@
+## Essential Dynamics Analysis using ProDy
+
+### Setup on the VSC
+
+1. Follow instructions on this page to install miniconda (or any conda version you want) and create a conda environment anywhere in your $VSC_DATA folder: [conda env on the VSC](https://docs.vscentrum.be/software/python_package_management.html#install-python-packages-using-conda)
+
+2. Install necessary packages
+```bash
+module load Python/3.7.0-foss-2018a
```

Comment:
Damn, that's a pretty old python. The minimum for chainsaw seems to be 3.8. We could use separate environments, I guess, it just seems wasteful. Is that the latest python version they support?